server/models/instance.py

```diff
@@ -71,6 +74,42 @@ def list(self, user=None, tale=None, limit=0, offset=0,
                 limit=limit, offset=offset):
             yield r
 
+    def updateAndRestartInstance(self, instance, token, imageId, digest):
+        """
+        Updates and restarts an instance.
+
+        :param image: The instance document to restart.
+        :type image: dict
+        :returns: The instance document that was edited.
+        """
+
+        instanceTask = update_container.signature(
+            args=[str(instance['_id'])], queue='manager',
+            kwargs={
+                'girder_client_token': str(token['_id']),
+                'image': str(kwargs['imageId']) + '@' + str(kwargs['digest'])
```

Comment:
```suggestion
                'image': str(imageId) + '@' + str(digest)
```

Reply:
Ack, somehow missed that one too... Thanks 😅